package.json

```diff
@@ -24,16 +24,18 @@
     "webpack-dev-server": "^3.11.2"
   },
   "dependencies": {
-    "css-loader": "^5.2.1",
-    "file-loader": "^6.2.0",
     "@babel/cli": "^7.13.14",
     "@babel/core": "^7.13.14",
     "@babel/preset-env": "^7.13.12",
     "@babel/preset-react": "^7.13.13",
     "babel-loader": "^8.2.2",
+    "bootstrap": "^4.6.0",
```

Comment:
Bootstrap is a pretty big dependency to bring in for a simple nav bar

Reply:
Are you even using this dependency? It doesn't look like it from the code.

Reply:
oh lol forgot to remove that, was trying out some stuff